concepticondata/conceptlists/Brid-2022-324.tsv

```diff
@@ -0,0 +1,325 @@
+Brid-2022-324-233	233	Heimia salicifolia				5420914	Heimia salicifolia Link	Plantae	Magnoliopsida	Heimia
+Brid-2022-324-234	234	Senna chloroclada				2957726	Senna chloroclada (Harms) H.S.Irwin & Barneby	Plantae	Magnoliopsida	Senna
+Brid-2022-324-235	235	Senna morongii				2957888	Senna morongii (Britton) H.S.Irwin & Barneby	Plantae	Magnoliopsida	Senna
+Brid-2022-324-236	236	Senna obtusifolia				2957408	Senna obtusifolia (L.) H.S.Irwin & Barneby	Plantae	Magnoliopsida	Senna
+Brid-2022-324-237	237	Senna occidentalis				2957966	Senna occidentalis (L.) Link	Plantae	Magnoliopsida	Senna
+Brid-2022-324-238	238	Senna pendula				2957187	Senna pendula (Willd.) H.S.Irwin & Barneby	Plantae	Magnoliopsida	Senna
+Brid-2022-324-239	239	Senna pendula var. eriocarpa				2957204	Senna pendula var. eriocarpa (Griseb.) H.S.Irwin & Barneby	Plantae		Senna
+Brid-2022-324-240	240	Senna sp.				2957478	Senna splendida (Vogel) H.S.Irwin & Barneby	Plantae		Senna
+Brid-2022-324-241	241	Bauhinia argentinensis				2953119	Bauhinia argentinensis Burkart	Plantae	Magnoliopsida	Bauhinia
+Brid-2022-324-242	242	ass, donkey	1862	DONKEY	asno, burro					
+Brid-2022-324-243	243	black	163	BLACK	negro					
+Brid-2022-324-244	244	white	1335	WHITE	blanco					
+Brid-2022-324-245	245	snail	1543	SNAIL	caracol					
+Brid-2022-324-246	246	foot	1301	FOOT	pie					
+Brid-2022-324-247	247	Tapirus terrestris				2440898	Tapirus terrestris (Linnaeus, 1758)	Animalia	Mammalia	Tapirus
+Brid-2022-324-248	248	flower	239	FLOWER	flor					
+Brid-2022-324-249	249	penis	1222	PENIS	pene					
+Brid-2022-324-250	250	Chauna torquata				5959222	Chauna torquata (Oken, 1816)	Animalia	Aves	Chauna
+Brid-2022-324-251	251	tongue	1205	TONGUE	lengua					
+Brid-2022-324-252	252	cow	1007	COW	vaca					
+Brid-2022-324-253	253	bear	1820	BEAR	oso					
+Brid-2022-324-254	254	chest	1592	CHEST	pecho					
+Brid-2022-324-255	255	ox	1169	OX	buey					
+Brid-2022-324-256	256	bull	1008	BULL	toro					
+Brid-2022-324-257	257	wasp	1517	WASP	avispa					
+Brid-2022-324-258	258	poison	655	POISON	veneno					
+Brid-2022-324-259	259	eyelash	1540	EYELASH	pestaña					
+Brid-2022-324-260	260	crocodile, alligator	1857	CROCODILE	cocodrilo, caimán					
+Brid-2022-324-261	261	heart	1223	HEART	corazón					
+Brid-2022-324-262	262	under	1485	BELOW OR UNDER	debajo					
+Brid-2022-324-263	263	every, all	98	ALL	cada uno, todo					
+Brid-2022-324-264	264	know	1410	KNOW (SOMETHING)	saber					
+Brid-2022-324-265	265	goose	1187	GOOSE	ganso					
+Brid-2022-324-266	266	cold	1287	COLD	frío					
+Brid-2022-324-267	267	nest	539	NEST	nido					
+Brid-2022-324-268	268	up, above	1591	UP	arriba, encima					
+Brid-2022-324-269	269	wake up	1602	WAKE UP	despertarse					
+Brid-2022-324-270	270	tapioca, manioc, cassava	925	CASSAVA	mandioca, yuca					
+Brid-2022-324-271	271	sweet potato	159	SWEET POTATO	camote, batata					
+Brid-2022-324-272	272	stand	570	STAND UP	poner(se) de pie					
```

Comment:
The English translation does not seem to be adequate. Do we talk about STAND or STAND UP here? Again, if in doubt, we should ask Nicolás.

Reply:
That's a good and complicated question. Are we talking about the difference between "there was a house standing nearby" and "the man stood up". The entry on IDS is "stand" and there's no "stand up". Moreover, at least in Spanish and Guaraní I'm sure the translation means "stand up" in the sense of "the man stood up". And when I searched in Spanish-based dictionaries, what I took into account was "pararse" (to stand up). Anyway, I'll have a closer look at the sources.

Reply:
So, the only argument in favor of STAND is actually IDS? In that case, I'd say the mapping is correct here and we adapt the English translation to "stand up". There are many inconsistencies with IDS translations that we should try to avoid.

Reply:
Then, I'll change the English translation and merge this PR if there are no further comments.

Reply:
I agree. Plus, in some of these languages there are verbs that mean, for example, "to be in a standing position", that is, "to stand". But since they are studied from a Spanish-language perspective, they are seldom confused with anything close to "stand up", because our language doesn't normally colexify those meanings :) For instance, in Maká the verbal root -n- means to be in a standing position. Its causative form means "to plant". But the verb meaning "to stand up" is totally unrelated, and that is clearly expressed in the dictionary.

Reply:
Even so, as I said, I'll double-check the entries, just in case.

Reply:
I'll rewrite my message more succintly. I agree with your decision. Even so, I'll double-check the entries on Edictor to make sure they really mean "stand up" and not "stand".

Reply:
Once I get your confirmation, I can also make the change on the dataset itself :)

Reply:
Confirmed. I just finished double-checking. Now we can be pretty sure that, at least if we trust the sources, all Edictor "stand" entries in Chaco languages actually mean "stand up". I had to delete some suspicious cases (Abipón, Tapiete, etc). Just to clarify, I refer to this Edictor file: 
http://lingulist.de/edictor/index.html?file=chacolanguages&remote_dbase=chacolanguages&columns=DOCULECT|SUBGROUP|CONCEPT|VALUE|FORM|TOKENS|ALIGNMENT|COGIDS|BORIDS|PATIDS|MORPHEMES|NOTE&basics=DOCULECT|CONCEPT|VALUE|FORM|TOKENS|COGID|COGIDS|MORPHEMES|NOTE&preview=100&async=true